docs/src/ice_dynamics.md

```diff
@@ -1,528 +0,0 @@
-# Halfar's model of glacial flow
```

Comment:
why did we delete this page?

Reply:
It looks like `docs\src\grigoriev.md` was errantly `git mv`’d to `docs\src\climate\ice_dynamics.md`, and the real `docs\src\ice_dynamics.md` was deleted.